.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: krew-index CI
+
+on: [push]
```

Comment:
Does this work on PRs? We need it to primarily work on PRs than pushes actually :)